### ISSUE TITLE
Loader is visible if the image has transparent/opaque areas

### ISSUE THREAD

**Issue by MicroDroid** [DCzajkowski/vue-pure-lightbox#11]

**Comment by DCzajkowski** [comment]
Yeah, it's possible. I don't know how to fix this though. I wanted to listen to some browser event when an image loaded, but I couldn't find anything. Any ideas?

**Comment by MicroDroid** [comment]
@DCzajkowski You can try to load it via an AJAX request. i.e. start with the thumbnail and blur it a bit, then load the full-size image in the background. Once done, replace the thumbnail with it.

**Comment by DCzajkowski** [comment]
Yeah, I don't think I'll be implementing that. If you want, you can fork this repo or make a PR 😉
